libs/ui-watt/src/lib/components/input/shared/watt-picker-base.ts

```diff
@@ -387,7 +387,7 @@ export abstract class WattPickerBase
   /**
    * @ignore
    */
-  private setSingleValue(value: string, input: HTMLInputElement) {
-    input.value = value;
+  private setSingleValue(value: string | undefined, input: HTMLInputElement) {
+    input.value = !value ? '' : value;
```

Comment:
suggestion: use default parameter instead:
```suggestion
  private setSingleValue(value = '', input: HTMLInputElement) {
    input.value = value;
```